Code/GraphMol/MolPickler.cpp

```diff
@@ -867,6 +868,15 @@ void MolPickler::_pickle(const ROMol *mol, std::ostream &ss,
     _pickleSSSR<T>(ss, ringInfo, atomIdxMap);
   }
 
+  // Write Stereo Groups
+  {
+    auto stereo_groups = mol->getStereoGroups();
```

Comment:
auto &stereo_groups, otherwise I believe you make a copy.

Reply:
thanks